AmplifyPlugins/Predictions/Tests/CoreMLPredictionsPluginUnitTests/DependencyTests/CoreMLNaturalLanguageAdapterTests.swift

```diff
@@ -53,6 +53,11 @@ class CoreMLNaturalLanguageAdapterTests: XCTestCase {
     ///    - I should get back correct tokens
     ///
     func testSyntaxToken() {
+        #if !os(visionOS)
+        // TODO: Test failing on visionOS.
```

Comment:
IIRC the CoreML tests are failing in all platforms, that's why they are not being run by the unit tests workflow. See https://github.com/aws-amplify/amplify-swift/pull/3655

Reply:
We couldn't really fix the tests because it was Xcode 15 on simulators that was broken. 
If the same issue remains in Xcode 16, I think we're on the same situation as before.